wasm_builder/src/lib.rs

```diff
@@ -58,21 +59,22 @@ pub fn make_message(investor_did: &InvestorDID, scope_did: &ScopeDID) -> [u8; 32
 ///   { "investor_did": [32_bytes_array], "investor_unique_id": [16_bytes_array] }
 ///
 /// # Errors
-/// * `TODO` panicing at the moment.
+/// * Failure to deserialize the cdd claim.
+/// * Failure to serialize the cdd id.
 #[wasm_bindgen]
-pub fn process_create_cdd_id(cdd_claim: String) -> String {
+pub fn process_create_cdd_id(cdd_claim: String) -> Result<String, JsValue> {
     let raw_cdd_data: RawCddClaimData = serde_json::from_str(&cdd_claim)
-        .unwrap_or_else(|error| panic!("Failed to deserialize the cdd claim: {}", error));
+        .map_err(|error| format!("Failed to deserialize the cdd claim: {}", error))?;
```

Comment:
The `JsValue` would have to be constructed with [`JsValue::from_str`](https://docs.rs/wasm-bindgen/0.2.68/wasm_bindgen/struct.JsValue.html#method.from_str), I believe.

An even better solution could be to define an error enum:
```
#[wasm_bindgen]
pub enum Error {
    ...
}
```
and change the return type to `Result<String, Error>`

Reply:
Tried that, did not work. It wanted me to implement another interface.